app/models/user.rb

```diff
@@ -1,6 +1,9 @@
 class User < ApplicationRecord
 	has_secure_password
 
+	 has_many :messages
+  has_many :chatrooms, through: :messages
```

Comment:
Weird indentation. Yesterday, we fixed this for Jules by ensuring his editor was set to use soft-tabs (spaces) instead of hard tabs.